app/src/main/java/com/jalasoft/routesapp/data/model/local/Location.kt

```diff
@@ -0,0 +1,6 @@
+package com.jalasoft.routesapp.data.model.local
+
+data class Location(
```

Comment:
Why not no use the Location from android.location?

Reply:
Because the Local Database (Room) don't recognize the android.location and can't save it